index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pl'>
+
+				<p>Jesteśmy społecznością osób, którym nie jest obojętny rozwój oraz stan światowej sieci internetowej oraz wyrażąmy <strong>głęboki niepokój</strong> w związku z technologią <a href='https://ampproject.org/'>Accelerated Mobile Pages</a> („AMP” – przyspieszone mobilne strony internetowe). AMP to projekt, którzy rzekomo ma na celu poprawę wygody użytkowników podczas korzystania z sieci internetowej.</p>
```

Comment:
~wyrażąmy~ → wyrażamy

Maybe "nie jest obojętny rozwój i stan" to not repeat "oraz".

I would also write "Przyśpieszone Mobilne Strony Internetowe" to indicate that it's a proprietary name.

~którzy rzekomo~ → który rzekomo